manifests/1.3.10/opensearch-dashboards-1.3.10-test.yml

```diff
@@ -0,0 +1,17 @@
+---
+schema-version: '1.0'
+name: OpenSearch Dashboards
+ci:
+  image:
+    name: opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v1
+components:
+  - name: OpenSearch-Dashboards
+    bwc-test:
+      test-configs:
+        - with-security
+        - without-security
+  - name: functionalTestDashboards
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
```

Comment:
We have changed the way we run dashboards test run so this should change to per component. @Divyaasm can add more.

Reply:
Any example? @gaiksaya @Divyaasm

Reply:
Example: https://github.com/Divyaasm/opensearch-build/blob/ghissue/manifests/2.6.0/opensearch-dashboards-2.6.0-test.yml
Not sure if the dashboards repo had backported the running in parallel part for 1.3 branch.

Reply:
Thanks, referring to that link and from what I understood we only need to add the components `opensearch-dashboards-1.3.10.yml` except for the `functionalTestDashboards`. Updated the change in my latest PR. However the backport of this PR https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/644 should be done to `1.3` to run the OSD integ tests.

Reply:
Right! This PR needs to be backported too https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/234